cockatrice/src/window_main.cpp

```diff
@@ -1031,6 +1031,11 @@ void MainWindow::actAddCustomSet()
         return;
     }
 
+    if (QFileInfo(fileName).suffix() != "xml") { // fileName = *.xml
```

Comment:
People can still override that, it's just a suggestion the opener can recommend (like Finder will say I can do "XML" or "All Files" )